conceptual/Caching/caching-pubsub.md

```diff
@@ -13,13 +13,15 @@ A typical answer to this issue is to use a centralized cache server (or a cluste
 
 An alternative solution to the problem of distributed caching is to have a local in-memory cache in each instance in the application. Instead of using a shared distributed cache, each application instance caches its own data into its own local cache. However, when one instance of the application modifies a piece of data, it needs to make sure that all instances remove the relevant items from their local cache. This is called *distributed cache invalidation*. It can be achieved easily and cheaply with a publish/subscribe (Pub/Sub) message bus such as Azure Service Bus, much less expensive than a cache cluster. 
 
-PostSharp allows you to easily add sub/sub cache invalidation to your existing PostSharp caching.
+PostSharp allows you to easily add pub/sub cache invalidation to your existing PostSharp caching.
 
 The principal inconvenience of pub/sub invalidation is that there is some latency in the invalidation mechanism, i.e. different instances of the application can see different data during a few dozens of milliseconds.
 
 
 ## Using Azure Service Bus pub/sub for distributed invalidation
 
+> [!WARNING]
+> `AzureCacheInvalidator` and `AzureCacheInvalidator2` are deprecated since PostSharp 2024.0. This guide covers the usage of <xref:PostSharp.Patterns.Caching.Backends.Azure.AzureServiceBusCacheInvalidator>.
```

Comment:
```suggestion
> `AzureCacheInvalidator` and `AzureCacheInvalidator2` are deprecated since PostSharp 2024.0. They are replaced by <xref:PostSharp.Patterns.Caching.Backends.Azure.AzureServiceBusCacheInvalidator>.
```